src/common/file.rs

```diff
@@ -38,19 +38,12 @@ pub fn put_file_contents(file: &str, contents: &[u8]) -> Result<(), std::io::Err
     Ok(())
 }
 
-#[inline(always)]
-pub fn delete_file(file: &str) -> Result<(), std::io::Error> {
-    std::fs::remove_file(file)?;
-    Ok(())
-}
-
```

Comment:
Why did you delete this function?

Reply:
AFAICS, this function is only ever used in this file's unit test. Functionality-wise, this is a wrapper over stdlib's `std::fs::remove_file`, it doesn't add anything new. Why would we want to create a level of indirection?

Am I missing some context?

Reply:
Yep, it's just a wrap for std lib and marked as `#[inline(always)] it will replace to std lib when compile. just want to provide a interface that you can create file, write file, delete file by common library.

Reply:
of cause, we don't use it anymore, we can delete it.